debian/changelog

```diff
@@ -1,3 +1,10 @@
+containerd.io (1.6.9-1) release; urgency=medium
+
+  * Update containerd to v1.6.9
+  * Update Golang runtime to 1.18.7
```

Comment:
Can you remind me how we actually make sure this change is correct? :see_no_evil:

Reply:
(as in, where is 1.18.7 actually specified?)

Reply:
It's picked up from the upstream dockerfile (and we have an override if needed); I may have to check that override in our internal pipeline, but it prints the version it uses during build, and I check that one 😁

Reply:
https://github.com/docker/containerd-packaging/blob/master/common/common.mk#L22

Reply:
All well 👍 

```
--------------------------------------------------------------------
Building packages on docker.io/library/ubuntu:focal

containerd   : v1.6.9 (commit: 1c90a44)
INFO: detected runc version (v1.1.4) from script/setup/runc-version
runc         : v1.1.4 (commit: 5fd4c4d)
architecture : x86_64
build image  : docker.io/library/ubuntu:focal
golang image : golang:1.18.7
--------------------------------------------------------------------
```